src/controllers/wishlistController.js

```diff
@@ -1,5 +1,34 @@
 const mongoose = require('mongoose');
 const Wishlist = require('../models/Wishlist');
+const { StatusCodes } = require('http-status-codes');
+
+//Create wishlist
+const createWishlist = async (req, res) => {
+  const { albumId } = req.body; //frontend sends the albumId in the request body
+
+  const existingWishlist = await Wishlist.findOne({ user: req.user.userId });
+
+  if (!existingWishlist) {
+    const newWishlist = new Wishlist({
+      user: req.user.userId,
+      albums: [], // start with an empty wishlist
+    });
+
+    if (albumId) {
+      newWishlist.albums.push(albumId);
+    }
```

Comment:
For now, since we're creating an empty wishlist, let's remove this block

Reply:
Done